card/mod/core/set/all/export.rb

```diff
@@ -1,42 +1,51 @@
 format :json do
-  def default_export_args args
-    args[:count] ||= 0
-    args[:count] += 1
-    args[:processed_keys] ||= ::Set.new
+  def default_export_args _args
+    @export_count ? @export_count += 1 : @export_count = 1
+    @exported_keys ||= ::Set.new
   end
 
-  # export the card itself and all nested content (up to 10 levels deep)
-  view :export, cache: :never do |args|
+  view :export, cache: :never do
     # avoid loops
-    return [] if args[:count] > 10 || args[:processed_keys].include?(card.key)
-    args[:processed_keys] << card.key
+    return [] if @export_count > 5 || @exported_keys.include?(card.key)
+    @exported_keys << card.key
+    Array.wrap(render_atom).concat(render_export_items).flatten
+  end
 
-    Array.wrap(render_atom(args)).concat(
-      render_export_items(count: args[:count])
-    )
+  def default_export_items_args _args
+    @exported_keys ||= ::Set.new
   end
 
-  def default_export_items_args args
-    args[:processed_keys] ||= ::Set.new
+  view :export_items, cache: :never do
+    valid_items_for_export.map do |item|
+      subformat(item).render_export
```

Comment:
`subformat`? not `nest`? In my experience `subformat` calls cause `voo` troubles

Reply:
I agree. I think I was just subtracting and subtracting from the old method (which used subformat).  will fix.